README.md

```diff
@@ -18,8 +18,8 @@ analysis and simulation.
 Current build status
 ====================
 
-Linux: [![Circle CI](https://circleci.com/gh/conda-forge/cdms2-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/cdms2-feedstock)
-OSX: [![TravisCI](https://travis-ci.org/conda-forge/cdms2-feedstock.svg?branch=master)](https://travis-ci.org/conda-forge/cdms2-feedstock)
+Linux: [![Circle CI](https://circleci.com/gh/conda-forge/cdms2-feedstock-dnadeau4-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/cdms2-feedstock-dnadeau4-feedstock)
```

Comment:
That doesn't look right

Reply:
I agree.  I used `conda smithy rerender -c auto` in my forked repository.

Reply:
Lets see what the webservices do